src/models/git/Branch.js

```diff
@@ -4,11 +4,12 @@
 class Branch {
   /**
    * Default constructor.
-   * @param {String} props.name - Branch name.
-   * @param {Boolean} [props.onLocal=false] - Indicate if branch is on local.
-   * @param {Boolean} [props.onRemote=false] - Indicate if branch is on remote.
-   * @param {String} [props.remote='origin'] - Remote value.
-   * @param {Boolean} [props.isCurrentBranch=false] - Indicates if this branch is the current.
+   * @param {string} props.name - Branch name.
+   * @param {boolean} [props.onLocal] - Indicate if branch is on local.
+   * @param {boolean} [props.onRemote] - Indicate if branch is on remote.
+   * @param {string} [props.remote] - Remote value.
+   * @param {boolean} [props.isCurrentBranch] - Indicates if this branch is the current.
+   * @param props
```

Comment:
put it at the top (before line 7)
@param {object} [props] - Object that contains all properties to set.

Reply:
done